sigmf/__init__.py

```diff
@@ -4,7 +4,7 @@
 #
 # SPDX-License-Identifier: LGPL-3.0-or-later
 
-__version__ = '1.1.2'
+__version__ = "1.1.3"
```

Comment:
This is a double increment -- current version is 1.1.1

Reply:
I pushed v1.1.2 to pypi yesterday.